lib/activerecord/all/activerecord.rbi

```diff
@@ -14,6 +14,48 @@ AssociationCallback = T.type_alias do
   )
 end
 
+class ActiveRecord::ConnectionAdapters::AbstractAdapter; end
+module ActiveRecord::QueryCache::ClassMethods; end
+module ActiveRecord::Querying; end
+module ActiveRecord::Translation; end
+module ActiveRecord::DynamicMatchers; end
+module ActiveRecord::Explain; end
+module ActiveRecord::AttributeAssignment; end
+module ActiveRecord::Integration; end
+module ActiveRecord::CounterCache; end
+module ActiveRecord::Attributes; end
+module ActiveRecord::AttributeDecorators; end
+module ActiveRecord::Locking::Optimistic; end
+module ActiveRecord::Callbacks; end
+module ActiveRecord::Timestamp; end
+module ActiveRecord::Aggregations; end
+module ActiveRecord::NoTouching; end
+module ActiveRecord::Reflection; end
+module ActiveRecord::Serialization; end
+module ActiveRecord::Store; end
```

Comment:
@jeffcarbs I feel like we should not add the static module definition and methods without signature to sorbet-typed. Those are usually handled by generated libraries like `rbi gems` or `tapioca`. Adding them to sorbet-typed increases the risk that there are conflicts between different tools.